libzeth/circuits/joinsplit.tcc

```diff
@@ -672,7 +612,7 @@ public:
         // The Merkle root is not in the `unpacked_inputs` so we subtract its
         // bit-length to get the total bit-length of the primary inputs in
         // `unpacked_inputs`
-        return get_inputs_bit_size() - FieldT::capacity();
+        return get_inputs_bit_size() - (1 + NumOutputs) * FieldT::capacity();
```

Comment:
You also subtract the length of the commitments here since they are not passed to the packers anymore. Please update the comment above accordingly

Reply:
Updated.